Assignment3/problem1.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+    secondLargest: function(array) {
+      var i,j,temp;
+      var n= array.length;
+      var first= array[0];
+      var second = array[0]
+      for(i=1;i<n;i++)
+        {
+          if(array[i]>first)
+            first=array[i];
+        }
+      for(i=1;i<n;i++)
+        {
```

Comment:
curly brace that starts any line should be on the same line.

`if (something) {`